packages/apptive_grid_theme/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://www.apptivegrid.de
 repository: https://github.com/ApptiveGrid/apptive_grid_flutter/tree/main/packages/apptive_grid_theme
 
 environment:
-  sdk: '>=2.17.0 <3.0.0'
+  sdk: '>=3.0.0 <4.0.0'
```

Comment:
I don't really see a reason why we would need to increase the version for the theme package

Reply:
I added this to keep all the packages on the same dart version. I can revert it, but to me it makes sense to keep the project on one version, just so we don't accumulate technical dept.

Reply:
Ok, I reverted the changes, but I would still keep the `refactor` tag, since we didn't add any new functionalities.